forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth
+
+import android.content.Context
+import android.content.SharedPreferences
+
+/**
+ * Listener to load the options from persistence for the given context
+ */
+interface ConfigInterface {
+    fun loadFromPreference(context: Context): FROptions
+}
+
+/**
+ * ConfigHelper Provide the helper methods to persist and load the configurations on start of the SDK
+ * This class also verify if the configuration changes on the current and previous launch.
+ */
+class ConfigHelper: ConfigInterface {
+
+    companion object {
+
+        //Alias to store Previous Configure Host
+        const val ORG_FORGEROCK_V_1_HOSTS = "org.forgerock.v1.HOSTS"
+
+
+        /**
+         * Persist the FROption Data in shared preference
+         *
+         * @param context  The Application Context
+         * @param frOptions Argument to persist the data in shared preference using the context
+         */
+        @JvmStatic
+        fun persist(context: Context, frOptions: FROptions) {
+            val sharedPreferences =
+                context.getSharedPreferences(ORG_FORGEROCK_V_1_HOSTS, Context.MODE_PRIVATE)
+            sharedPreferences.edit().putString("url", frOptions.server.url)
+                .putString("realm", frOptions.server.realm)
```

Comment:
Strings like "realm", "cookieName" etc are used multiple times. Could we move them to an enum or statics to avoid re-write and copy paste errors?

Reply:
good point